src/test/java/org/junitpioneer/platform/testkit/engine/PioneerEngineExecutionResults.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.platform.testkit.engine;
+
+import static java.util.stream.Collectors.toList;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.junit.platform.engine.TestExecutionResult;
+import org.junit.platform.engine.discovery.DiscoverySelectors;
+import org.junit.platform.testkit.engine.EngineExecutionResults;
+import org.junit.platform.testkit.engine.EngineTestKit;
+import org.junit.platform.testkit.engine.Events;
+
+/**
+ * Pioneers own class to handle {@link org.junit.platform.testkit.engine.EngineExecutionResults} of the JUnit-Jupiter-Engine.
+ *
+ */
+public class PioneerEngineExecutionResults {
+
+	EngineExecutionResults executionResults;
+
+	public PioneerEngineExecutionResults(Class<?> testClass) {
+		executionResults = EngineTestKit
+				.engine("junit-jupiter")
+				.selectors(DiscoverySelectors.selectClass(testClass))
```

Comment:
Consider static importing `DiscoverySelectors` methods. Would that be more readable?

Reply:
Personally I like regular imports in such cases more as it's clear that this messages does not belong to the class. But if the majority wants to use static imports I can switch.

Reply:
I like static imports in general, but I prefer them in situations where a somewhat universal API is used that is frequently seen. `DiscoverySelectors::selectClass` doesn't quite qualify, but I'm still ok, either way.

Reply:
I would like to keep them. So I resolve this conversation.